Examples/DigiMeSDKExample/DigiMeSDKExample/ViewController.swift

```diff
@@ -49,7 +49,17 @@ wtEHEfBLVfL+dCrCpgjLOb18ZXpFM2LuX1+mB347D3d5k8OnHY0hrfZZI5x8hQFT
 
         let config = Configuration(appId: appId, contractId: contractId, privateKey: privateKey)
 
-        sdk = DigiMeSDK(configuration: config)
-        sdk?.testNewUser()
+        sdk = DigiMeSDK.DigiMe(configuration: config)
```

Comment:
I don't think so. I renamed the class `DigiMeSDK` to `DigiMe` because I thought devs should be able to distinguish classes from different modules by prepending with module name. However when I did  `DigiMeSDK.DigiMeSDK(...)` the compiler threw an error.